dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -57,32 +61,43 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
         }
 
         log.info("Retrieving all the forks...");
-        PagedIterable<GHRepository> repos = dockerfileGitHubUtil.getGHRepositories(parentToPath, currentUser);
+        PagedIterable<GHRepository> listOfcurrUserRepos =
+                dockerfileGitHubUtil.getGHRepositories(pathToDockerfilesInParentRepo, currentUser);
 
-        String message = ns.get("m");
         List<IOException> exceptions = new ArrayList<>();
-        for (GHRepository repo : repos) {
+        List<String> skippedRepos = new ArrayList<>();
+
+        for (GHRepository currUserRepo : listOfcurrUserRepos) {
             try {
-                changeDockerfiles(parentToPath, parentToImage, imageToTagMap, repo, message, ns.get("c"));
+                changeDockerfiles(ns, pathToDockerfilesInParentRepo, imagesFoundInParentRepo, imageToTagMap, currUserRepo,
+                        skippedRepos);
             } catch (IOException e) {
                 exceptions.add(e);
             }
         }
 
-        if (exceptions.size() > 0) {
+        if (!exceptions.isEmpty()) {
```

Comment:
Positive conditions are generally more readable

Reply:
Yep agreed but in this case, isn't `isEmpty()` [more readable/preferred](https://stackoverflow.com/a/27525511/2891840) than `size() > 0`?

Reply:
I'll leave it alone; not going to block things on nitpicks :)